AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/related.js

```diff
@@ -70,6 +70,16 @@ define([
             return record.series;
         },
 
+        /**
+         * Count number of images in Related Serie
+         *
+         * @param {Object} record
+         * @returns boolean
+         */
+        countSeriesImages: function(record) {
+            return (parseInt(record.series().length, 10) >= 4);
```

Comment:
Radix parameter is 10 by default, so it's possible to skip passing this optional parameter to `parseInt`. Also, braces are not necessary.

```suggestion
            return parseInt(record.series().length) >= 4;
```